components/sysutils/clamav/Makefile

```diff
@@ -23,7 +23,7 @@ COMPONENT_NAME=		clamav
 ### But our IPS version should better have same number of components.
```

Comment:
You should add
USE_COMMON_TEST_MASTER= no
to the Makeflle in order make the tests pass.